packages/client/logic/nav.ts

```diff
@@ -45,30 +60,10 @@ export const currentLayout = computed(() => currentRoute.value?.meta?.layout ||
 export const nextRoute = computed(() => rawRoutes.find(i => i.path === `${Math.min(rawRoutes.length, currentPage.value + 1)}`))
 export const prevRoute = computed(() => rawRoutes.find(i => i.path === `${Math.max(1, currentPage.value - 1)}`))
 
-export const clicksElements = computed<HTMLElement[]>(() => {
-  // eslint-disable-next-line no-unused-expressions
-  routeForceRefresh.value
-  return currentRoute.value?.meta?.__clicksElements || []
-})
-
-export const clicks = computed<number>({
-  get() {
-    if (isClicksDisabled.value)
-      return 99999
-    let clicks = +(queryClicks.value || 0)
-    if (Number.isNaN(clicks))
-      clicks = 0
-    return clicks
-  },
-  set(v) {
-    queryClicks.value = v.toString()
-  },
-})
-
-export const clicksTotal = computed(() => +(currentRoute.value?.meta?.clicks ?? clicksElements.value.length))
+export const clicks = computed(() => usePrimaryClicks(currentRoute.value))
```

Comment:
Let's name it `clicksContext` and bring back to number version of `clicks` - there are slides are still using `$slidev.nav.clicks` (it was the only way before we have `$clicks`)